tldraw/tldraw
# Make some missing tsdocs appear on the docs site

🚨 Note 🚨
This PR has changed! See my [newer comment](https://github.com/tldraw/tldraw/pull/1706#issuecomment-1623451709) for what the PR does now.
This description is kept here to show the original intention of the PR.

---

This PR fixes the tsdocs formatting of `TldrawEditorProps`, so that they appears on the docs site.

We have docs already written, but they weren't appearing. There are probably others like this too.

![image](https://github.com/tldraw/tldraw/assets/15892272/8d8940b3-983f-48b3-9804-7ac88116ca9d)

### Change Type


[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Test Plan

1. Navigate to `/gen/editor/TldrawEditorProps`
2. Make sure that that the parameters are listed out with descriptions.


### Release Notes

- Docs: Fixed some missing docs for the TldrawEditor component.
